rest/tools/oshinko-test.sh

```diff
@@ -75,20 +75,47 @@ while getopts :t:p:r:b:n:i:h opt; do
     esac
 done
 
+# Where this script lives ...
+SCRIPT_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"`)
+
 case "$REQUESTED_TEST" in
     unit)
         # bring in the tag variable
-        . tools/common.sh
+        source $SCRIPT_DIR/common.sh
         # this export is needed for the vendor experiment for as long as go
         # version 1.5 is still in use.
         go get gopkg.in/check.v1
         export GO15VENDOREXPERIMENT=1
         go test -v -ldflags "$TAG_APPNAME_FLAGS" "github.com/radanalyticsio/oshinko-cli/rest/tests/unit"
         ;;
 
+    client-local)
+        PROJECT=$(oc project -q)
+        export OSHINKO_CLUSTER_NAMESPACE=$PROJECT
+        export OSHINKO_KUBE_CONFIG=~/.kube/config
+        set +e
+        oc create sa oshinko -n $PROJECT
+        oc policy add-role-to-user admin system:serviceaccount:$PROJECT:oshinko -n $PROJECT
+
+        # These empty configmaps are needed for tests that look at reported config
+        # Since they're empty they're not actually used, just reported back in status, this is fine
+        oc create configmap clusterconfig
```

Comment:
Well, strictly speaking, that stuff doesn't come in until https://github.com/radanalyticsio/oshinko-cli/pull/85 is merged. But no harm done